lib/questions.rb

```diff
@@ -1,61 +1,82 @@
 # keep only the elements that start with an a
 def select_elements_starting_with_a(array)
+  array.select{|array| array[0] == "a"}
 end
 
 # keep only the elements that start with a vowel
 def select_elements_starting_with_vowel(array)
+  array.select {|x| ["a","e","i","o","u"].include? x[0]}
 end
 
 # remove instances of nil (but NOT false) from an array
 def remove_nils_from_array(array)
+  # array.delete(nil)
+  array.reject {|item| item == nil}
 end
 
 # remove instances of nil AND false from an array
 def remove_nils_and_false_from_array(array)
+    array.reject {|item| item == nil || item == false}
```

Comment:
Use 2 (not 4) spaces for indentation.<br>Space between { and | missing.<br>Space missing inside }.